src/rooted_gradient_transforms.cpp

```diff
@@ -185,3 +239,19 @@ EigenVectorXd RatioGradientOfHeightGradientEigen(
   }
   return eigen_output;
 }
+
+// NOTE: This is moved from fat_beagle.cpp, at FatBeagle::LogDeterminantJacobian.
```

Comment:
We can see the diff 😄 let's drop this comment.

Reply:
Yep.  Removed.